.jenkins/Jenkinsfile

```diff
@@ -420,12 +420,12 @@ try{
             "Sim 1804 clang-7 SGX1 Release" :                          { simulationTest('18.04', 'SGX1', 'Release')},
             "Sim 1804 clang-7 SGX1-FLC Debug" :                        { simulationTest('18.04', 'SGX1FLC', 'Debug')},
             "Sim 1804 clang-7 SGX1-FLC Release" :                      { simulationTest('18.04', 'SGX1FLC', 'Release')},
-            // "RHEL-8 simulation clang-8 SGX1 Release":             { RHEL8Test('clang', 'Release', ["OE_SIMULATION=1"]) }, // Enable when https://github.com/openenclave/openenclave/issues/2556 is fixed.
+            "RHEL-8 simulation clang-8 SGX1 Release":                  { RHEL8Test('clang', 'Release', ["OE_SIMULATION=1"]) },
```

Comment:
Issue #2556 is not listed as being resolved so this may still fail.

Reply:
Thanks for pointing this out!